src/action/head_to_goal_control/head_to_goal_control/controller.py

```diff
@@ -19,16 +21,20 @@ def __init__(self):
 
         # Constant to tune (touch me please it makes me feel horny ahhhhhhh!)
         ## Tuning for look ahead distance
-        self.look_up_distance = 2
-        self.cancel_distance = 0.5
+        self.look_up_distance = 5
+        self.cancel_distance = 2
         ## Tuning for PID controller
-        self.P = 40
+        self.P = 20
         self.max_steering_angle = 20.0
         #self.max_speed = 2.5
-        self.max_speed = 4.0
+        self.max_speed = 2
         self.speed_adjuster_width = 15
         ## Current speed setting
-        self.current_speed = 0
+        self.current_speed = 0.00
+        self.desired_speed = 0.00
+
+        ## if using RL, set to True
+        self.usingRL = False
```

Comment:
make the constants a CONST at the top of the file or to be passed in as a config